docs/slides/innersource-summit.md

```diff
@@ -0,0 +1,237 @@
+# The InnerSource Program Office (ISPO) Working Group
+<!---  Slide 2: Opening Statement --->
+
+![bg right](../assets/innersource-commons.svg)
+
+The InnerSource Program Office (ISPO) Working Group develops guidance and products for enterprise InnerSource teams, not individuals or projects.
+
+<!--
+1. Define an ISPO
+2. ...
+-->
+--- 
+<!--- Slide 3 --->
+
+For those with InnerSource responsibilities in an OSPO, platform engineering, or other centralized group, this working group is for you.
+
```

Comment:
1. Added an example of other centralized groups.
2. Broke out into a list to make it easier for readers to identify themselves in the slide.

Do we want to add a note here to keep people out of these groups engaged? 

Alternatively, this is a quick "check out" cue to save people time.

```suggestion
If you're in any of these groups, we invite you to join us:

* InnerSource responsibilities in an Open Source Program Office (OSPO)
* Platform Engineering Teams
* Other centralized groups such as Enterprice Architecture
```